week-1/interpret/percentage-change.js

```diff
@@ -1,22 +1,32 @@
 let carPrice = "10,000";
 let priceAfterOneYear = "8,543";
-
-carPrice = Number(carPrice.replaceAll(",", ""));
-priceAfterOneYear = Number(priceAfterOneYear.replaceAll(",", ""));
+carPrice = Number(carPrice.replace(/,/g, ""));
+priceAfterOneYear = Number(priceAfterOneYear.replace(/,/g, ""));
 
 const priceDifference = carPrice - priceAfterOneYear;
 const percentageChange = (priceDifference / carPrice) * 100;
 
 console.log(`The percentage change is ${percentageChange}`);
 
+
+
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
+// 2 functions are called in this code. `Number()` and `replaceAll()` each two times has have been called .
 
 // b) Run the code and identify the line where the error is coming from - why is this error occurring? How can you fix this problem?
+//the error happens when the code runner encounter replaceAll() method . i tried different ways but it still doesn't work and 
```

Comment:
Which code runner are you trying to use for this? Are you referring to the Python visualiser?